src/test/java/edu/hm/hafner/analysis/assertj/IssueAssert.java

```diff
@@ -0,0 +1,136 @@
+package edu.hm.hafner.analysis.assertj;
+
+import java.util.Objects;
+import java.util.UUID;
+
+import org.assertj.core.api.AbstractAssert;
+
+import edu.hm.hafner.analysis.Issue;
+import edu.hm.hafner.analysis.Priority;
+
+public class IssueAssert extends AbstractAssert<IssueAssert, Issue> {
+    public IssueAssert(final Issue actual) {
+        super(actual, IssueAssert.class);
+    }
+
+    public static IssueAssert assertThat(final Issue actual) {
+        return new IssueAssert(actual);
+    }
+
+    public IssueAssert hasId(final UUID id) {
+        isNotNull();
+
+        if (!Objects.equals(actual.getId(), id)) {
+            failWithMessage("Expected issue's id to be <%s> but was <%s>", id, actual.getId());
```

Comment:
private static final String EXPECTED_BUT_WAS_MESSAGE = "%nExpecting:%n <%s>%nto be equal to:%n <%s>%nbut was not.";